elements/bulbs-slick-slideshow/bulbs-slick-slideshow.js

```diff
@@ -109,10 +113,15 @@ class BulbsSlickSlideshow extends BulbsHTMLElement {
 
   slideshowChanged (event, slickObject, currentSlide) {
     window.location.hash = currentSlide;
-    window.onionan.trackPageView(true);
+    this.trackPageView();
     this.enableDisableNav(this.slides, currentSlide);
   }
 
+  trackPageView () {
+    this.analyticsManager.setDimension('navigation_method','slideshow');
+    this.analyticsManager.trackPageView(false);
```

Comment:
We may want to also add some protection into this method, to at least check for the presence of `this.analyticsManager` and log an error if it's not there